src/atomics/poly.jl

```diff
@@ -1,27 +1,39 @@
-
-function poly(scene::makie, points, attributes::Dict)
-    last(points) == first(points) && pop!(points)
-    polys = GeometryTypes.split_intersections(points)
-    result = []
-    for poly in polys
-        mesh = GLNormalMesh(poly) # make polygon
-        if !isempty(GeometryTypes.faces(mesh)) # check if polygonation has any faces
-            push!(result, GLVisualize.visualize(mesh, Style(:default), kw_args))
-        else
-            warn("Couldn't draw the polygon: $points")
-        end
+function _poly(scene::makie, attributes::Dict)
+    attributes = poly_defaults(scene, attributes)
+    println(typeof(attributes[:positions]))
+    bigmesh = lift_node(attributes[:positions]) do p
+        polys = GeometryTypes.split_intersections(p)
+        merge(GLPlainMesh.(polys))
     end
-    result
+    mesh(scene, bigmesh, color = attributes[:color])
```

Comment:
So this basically does the `fill` part, right?

Reply:
exactly!